essential.md

```diff
@@ -243,20 +247,47 @@ The `Iptc4xmpExt:PersonShownInTheImage` shall contain at least one of the follow
 
 | Field | Type | Stores |
 | :---- | :--- | :----- |
-| `mwg-rs:Regions` | `nested elements` | list of coordinates and names for face tags in image |
-
+| `mwg-rs:Regions` | nested elements | one RegionList |
+| `mwg-rs:RegionList` | `rdf:Bag` | one more more Regions |
+| `mwg-rs:Type` | Closed set | FHMWG supports `Face`; mwg-rs also allows `Pet`, `BarCode`, and `Focus` |
+| `mwg-rs:Title` | Line string | A short descriptive title of the contents of the region; preferably the name of the person |
+| `mwg-rs:Description` | Block string | A longer description of the contents of the region |
+| `mwg-rs:Type` | Closed set | FHMWG supports `Face`; mwg-rs also allows `Pet`, `BarCode`, and `Focus` |
+| `mwg-rs:Area` | nested elements | a region as a center point and extents |
+| `stArea:unit` | Closed set | always the string `normalized` |
+| `stArea:x` | Number | Center of region, in relative horizontal units between 0 and 1 |
+| `stArea:y` | Number | Center of region, in relative vertical units between 0 and 1 |
+| `stArea:w` | Number | Width of region, in relative horizontal units between 0 and 1 |
+| `stArea:h` | Number | Height of region, in relative vertical units between 0 and 1 |
+| `stArea:d` | Number | Diameter of region, in the smaller of the units of `stArea:w` and `stArea:h` |
 
 
 #### 2.6.2. Details   <a name="2.6.2"></a>
 
-The `mwg-rs:Regions` shall contain at least one of the following:
-
-- 
-- 
-- 
+Face regions are encoded as
 
+- The top-level `rdf:Description`
+- shall contain 0 or 1 `mwg-rs:Regions`
+- which shall contain 1 `mwg-rs:RegionList`
+- which shall contain 1 `rdf:Bag`
+- which shall contain 1 or more `rdf:li` with `rdf:parseType="Resource"`
+- which shall contain all of the following:
+    - `mwg-rs:Type` with line string value "`Face`"
```

Comment:
I recommend we limit the face tag to a rectangle.  Although a point and circle are both legal areas, most applications use rectangle coordinates (even if they are displaying circles.  If we limit the implementation to rectangles, it makes it easier for applications to read and display the face tags.

Additionally, the X,Y definition is not clear for "center of rectangle" -- In practice, the X,Y for a rectangle is the upper left corner.  "Center of rectangle" implies something different for me.